packages/gatsby-theme-store/src/sdk/search/useAutocompleteSuggestions.ts

```diff
@@ -25,10 +23,7 @@ export const useAutocompleteSearchSeggestions = ({ term }: Props) => {
     suspense: true,
   })
 
-  return {
-    query,
-    ...context,
-  }
+  return data!.vtex.autocompleteSearchSuggestions!.searches!
```

Comment:
How do you know that values always will exist?

Reply:
Because if they don't, this means something went wrong with the query and I'll get and `errors` array in the graphql response. 

The graphql plugin is not generating the correct typings in this case